tests/clihighlight.txt

```diff
@@ -915,31 +915,31 @@
       c[0m
     [37mGROUPS[0m
       [37mBETWEEN[0m [37mUNBOUNDED[0m [37mPRECEDING[0m
-      [37mAND[0m [37mCURRENT ROW[0m
-      [37mEXCLUDE[0m [37mNO OTHERS[0m
+      [37mAND[0m CURRENT[0m [37mROW[0m
+      [37mEXCLUDE[0m NO[0m OTHERS[0m
```

Comment:
That line contains `[0m` three times in a row… that's not valid, is it?

Reply:
IDK, in any case, this is unrelated to this PR, previously two reserved words were easily tokenized as well.

Reply:
Previously, there wasn't two consecutive `[0m` without a `[37m` in between.

Reply:
To answer your initial question, all should be fine, `[37m` sets "light gray color" and `[0m` resets it.

The `CURRENT` keyword was not present before but is reserved and it will be fixed in https://github.com/doctrine/sql-formatter/pull/106. Can this PR be merged as is or do you want me to port the `CURRENT` keyword into this PR?

Currently this is printed as:
![image](https://github.com/doctrine/sql-formatter/assets/2228672/dc8a9e22-a2b0-4832-95e2-6cbad7c382eb)

I prefer to fix this in the #106 PR as other keywords (like `NO`, `OTHERS`, ...) were/are not highlighted as keywords either.

Reply:
Fine I guess.